### DISCUSSION TITLE
Hosting in same code base as Wordpress install?

### DISCUSSION THREAD

**Discussion by hamburger-fries** [prappo/wordpress-plugin-boilerplate#7] [Q&A]
Do you host in the same code base as the WordPress site? Or is this hosted in something like Vercel?
I know Wordpress Core has a BUILD process.

I love love what your have built. I like Tailwind and use Flowbite alot.

**Comment by prappo** [comment]
@forestcall  This WordPress plugin boilerplate code is hosted using https://wordpress.org/playground/
